tests/functional/eventbridge/test_put_events.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import base64
+import json
+import os
+import tempfile
+from datetime import datetime
+from unittest import mock
+
+from awscli.compat import urlparse
+from awscli.testutils import BaseAWSCommandParamsTest
+
+
+class TestPutEventsCommand(BaseAWSCommandParamsTest):
+    def setUp(self):
+        super().setUp()
+        self.access_key = "ABCDEFGHIJKLMNOPQRST"
+        self.secret_key = "TSRQPONMLKJUHGFEDCBA"
+        self.session_token = "TOKENTOKENTOKENTOKEN"
+        self.environ["AWS_ACCESS_KEY_ID"] = self.access_key
+        self.environ["AWS_SECRET_ACCESS_KEY"] = self.secret_key
```

Comment:
We should not be needing to define a `setUp` as the [`BaseAWSCommandParamsTest`](https://github.com/aws/aws-cli/blob/5fb687de952f312f2b9ee19f226afc0975b56e94/awscli/testutils.py#L343) should be already patching the environment vars out for us.